database/factories/FieldFactory.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Faker\Generator as Faker;
+
+$factory->define(\App\Models\Field::class, function (Faker $faker) {
+    return [
+        'name' => $faker->name,
+        'type' => $faker->randomElement(['text', 'textarea']),
+        'form' => 'kitchen'
```

Comment:
Use the classname (Kitchen::class) here instead

Reply:
ok